gpt4all-chat/translations/gpt4all_ro_RO.ts

```diff
@@ -640,543 +799,423 @@
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="75"/>
-        <source>GPT4All</source>
-        <translation type="unfinished">GPT4All</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="75"/>
-        <source>You</source>
-        <translation type="unfinished">Tu</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="97"/>
-        <source>response stopped ...</source>
-        <translation type="unfinished">replică întreruptă...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="98"/>
-        <source>retrieving localdocs: %1 ...</source>
-        <translation type="unfinished">se preia din LocalDocs: %1 ...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="99"/>
-        <source>searching localdocs: %1 ...</source>
-        <translation type="unfinished">se caută în LocalDocs: %1 ...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="100"/>
-        <source>processing ...</source>
-        <translation type="unfinished">procesare...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="101"/>
-        <source>generating response ...</source>
-        <translation type="unfinished">se generează replica...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="102"/>
-        <source>generating questions ...</source>
-        <translation type="unfinished">se generează întrebări...</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="212"/>
-        <location filename="../qml/ChatItemView.qml" line="610"/>
-        <source>Copy</source>
-        <translation type="unfinished">Copiere</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="218"/>
-        <source>Copy Message</source>
-        <translation type="unfinished">Copiez mesajul</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="228"/>
-        <source>Disable markdown</source>
-        <translation type="unfinished">Dezactivez markdown</translation>
-    </message>
-    <message>
-        <location filename="../qml/ChatItemView.qml" line="228"/>
-        <source>Enable markdown</source>
-        <translation type="unfinished">Activez markdown</translation>
-    </message>
-    <message numerus="yes">
-        <location filename="../qml/ChatItemView.qml" line="355"/>
-        <source>%n Source(s)</source>
-        <translation type="unfinished">
-            <numerusform>%n Sursa</numerusform>
-            <numerusform>%n Surse</numerusform>
-            <numerusform>%n de Surse</numerusform>
-        </translation>
-    </message>
```

Comment:
Why are parts like this being deleted? This will cause the `N Sources` text on a response with attached LocalDocs sources to be untranslated.